src/pages/Vans.js

```diff
@@ -28,6 +35,32 @@ export default function Vans() {
   return (
```

Comment:
Why is there not some type of loop here that introspects the data as returned from the API call (the returned data from the mock) that builds the buttons from that array or object (the returned json) so that when the data changes its reflected in the output? Maybe I am missing something but since you are returning it hard coded I take it that is the way it's being built. I do not see much point in that. It is not real world usage.